pdc/apps/common/viewsets.py

```diff
@@ -200,7 +200,8 @@ def initial(self, request, *args, **kwargs):
             self._check_ordering_keys(request)
 
     def _check_ordering_keys(self, request):
-        ordering_keys = request.query_params.get('ordering')
+        # Allow ordering to be defined multiple times and also as comma separated list.
+        ordering_keys = ','.join(request.query_params.getlist('ordering'))
```

Comment:
The more common way is to do `ordering=aaa&ordering=bbb`. In this case `request.query_params.getlist('ordering')` returns `["aaa", "bbb"]` but `request.query_params.get('ordering')` returns just `bbb` but both keys are used sorting.

This also happens when using [pdc client library](https://github.com/product-definition-center/pdc-client/blob/f0b60a38193332d72c4a21ca345dd726d736eb80/pdc_client/plugins/release_variant.py#L104).